debug_module.tex

```diff
@@ -84,6 +84,15 @@ \section{Reset Control} \label{reset}
 Otherwise, if the hart was initially running it will execute normally (running state)
 and if the hart was initially halted it should now be running but may be halted.
 
+To determine when a requested \FdmDmcontrolNdmreset system reset has
+completed, the debugger can poll the \FdmDmstatusNdmresetpending bit,
+if implemented.  To allow for detection of this optional feature,
+\FdmDmstatusNdmresetpending, if implemented, returns one on reads whenever
+\FdmDmcontrolNdmreset is one.  If implemented, it remains one after deassertion of
+\FdmDmcontrolNdmreset until the requested system reset is complete,
+after which it is cleared by the Debug Module.  If this feature is not implemented, 
+\FdmDmstatusNdmresetpending always returns zero.
```

Comment:
Do we need this? It seems to say exactly what is said in the definition of ndmresetpending, but in more words. If you want to highlight the detection mechanism, I'd add a sentence in that definition about it.

Reply:
removed unneeded text in new git push.